### ISSUE TITLE
AttributeError: 'pymc3' object has no attribute 'switch'

### ISSUE THREAD

**Issue by vladserkoff** [CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers#319]
Hey, thanks for the transition to pymc3!
There's probably a typo in the 8th cell in [Intro Chapter](https://github.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers/blob/master/Chapter1_Introduction/Ch1_Introduction_PyMC3.ipynb)
Should `lambda_ = pm.switch(tau >= idx, lambda_1, lambda_2)` be `lambda_ = pm.math.switch(tau >= idx, lambda_1, lambda_2)` instead?

**Comment by twiecki** [comment]
Thanks! CC @mmargenot

**Comment by mmargenot** [comment]
Yeah, that looks like it is due to a more recent change in the package.

Both
`lambda_ = pm.math.switch(tau >= idx, lambda_1, lambda_2)`

and

```import theano.tensor as tt
lambda_ = tt.switch(tau >= idx, lambda_1, lambda_2)```

should work.

**Comment by twiecki** [comment]
Closed by #321 .

**Comment by vladserkoff** [comment]
Thanks!
